support/k8s/k8s-workload-registrar/config_crd.go

```diff
@@ -149,11 +191,33 @@ func (c *CRDMode) Run(ctx context.Context) error {
 	return mgr.Start(ctrl.SetupSignalHandler())
 }
 
-func getNamespace() (string, error) {
-	content, err := ioutil.ReadFile(namespaceFile)
+func (c *CRDMode) certDirExistsAndReadOnly() (bool, error) {
```

Comment:
nit: this would be a little more robust if we did not do the stat and instead handled the proper error condition returned from `unix.Access`:
```
func (c *CRDMode) certDirExistsAndReadOnly() (bool, error) {
	err := unix.Access(c.WebhookCertDir, unix.W_OK)
	switch {
	case err == nil, errors.Is(err, unix.ENOENT):
		return false, nil
	case errors.Is(err, unix.EROFS):
		return true, nil
	default:
		return false, err
	}
}
```

Reply:
Thanks for the suggestion. This change is in.